docs/toc.md

```diff
@@ -100,7 +100,7 @@
 ### [F# unit testing with dotnet test and xUnit]()
 ### [F# unit testing with dotnet test and MSTest]()
 ### [Running selective unit tests](core/testing/selective-unit-tests.md)
-### [Live unit testing .NET Core projects with Visual Studio]()
+### [Live unit testing .NET Core projects with Visual Studio](https://docs.microsoft.com/en-us/visualstudio/test/live-unit-testing-start?tabs=csharp)
```

Comment:
it should be a relative path like the following to work in offline book and localized versions:
/visualstudio/test/live-unit-testing-start

Reply:
Ah, I wasn't sure how to do that with a link outside this repo. Thank you!